extensions/markdown-language-features/src/languageFeatures/copyFiles/shared.ts

```diff
@@ -141,28 +204,46 @@ interface UriListSnippetOptions {
 	readonly separator?: string;
 }
 
+export function createLinkSnippet(
```

Comment:
It's a little weird to have this function both take and return a `SnippetString`.Can you just have it return a new `SnippetString` that it creates?